cmd/operator/deploy/operator/10-collector.yaml

```diff
@@ -115,10 +115,10 @@ spec:
           value: "25"
         resources:
           limits:
-            memory: 3G
+            memory: 2G
```

Comment:
It would be quite amazing to depend that to machine type.. do we have plans for it? (TODO/ticket is fine)

Reply:
@bwplotka can you elaborate on how memory usage requirements might vary across different machine types?

Reply:
I'm honestly not sure how you would do this dynamically other than maybe using a [VPA](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler)?